docs/modules/user-guide/partials/proc_adding-a-volume-component-to-a-devfile.adoc

```diff
@@ -13,22 +13,9 @@ You can use a volume component to share files among container components and col
 .Procedure
 
 . Define a `volume` component in your devfile and specify a volume size for it.
-+
-.A minimal `volume` component
```

Comment:
Do you intend to delete just this line?   The example you had before matched `Define a `volume` component in your devfile and specify a volume size for it.`